src/pybind/mgr/prometheus/module.py

```diff
@@ -1577,6 +1577,20 @@ def get_pool_repaired_objects(self) -> None:
 
             self.metrics[path].set(stats['stat_sum']['num_objects_repaired'],
                                    labelvalues=(stats['poolid'],))
+    
+    def get_all_daemon_health_metrics(self):
+        daemon_metrics = self.get_daemon_health_metrics()
```

Comment:
Just curious, is this a metric that will/should be moved to the ceph-exporter?

Reply:
It is possible to extend the admin_socket to expose a new endpoint where you can parse the same output. We would have to extract the functionality from the `ActivePyModules.cc` and add the formatting capabilities to the DaemonHealthMetric class itself to easily translate the list of metrics in the socket.